app/raid.js

```diff
@@ -36,6 +36,7 @@ class Raid extends Party {
     if (!raidExists) {
       // add some extra raid data to remember
       raid.createdById = memberPrivacy ? -1 : memberId;
+      raid.originallyCreatedBy = memberId;
```

Comment:
Make sure you add this field in the toJSON() method as well so it doesn't get lost across bot restarts

Reply:
Should this be in `Raid.toJSON()` or `Party.toJSON()`? Seems to over lap somewhat. I'm leaning towards `Raid`...

Reply:
Right now it'd belong in raid - but I could see putting it in party instead if the privacy stuff is extended for all of the possible things reported (currently only raid trains, and the other type that doesn't yet exist is generic meetups).  But at the same time, I'm not sure that reporting either of those anonymously makes a ton of sense. 😛

Reply:
I can see why reporting a raid anonymously might be a thing for some overly private people. For example, if I don't participate and say !interested, !complete, !here, etc., nobody in the channel knows i'm there but I can still play and go to raids based on the channels. If I don't wnat people to know where I am, I'm unlikely to report raids I see because it will always say "reported by kingkovifor" and than anyone knows I'm currently or was recently in the area.

Meet ups wouldn't necessarily make sense to report anonymously. But give me 30s and that'll be up. lol.